tests/suites/test_suite_pkparse.data

```diff
@@ -912,34 +912,68 @@ Parse Public EC Key #2 (RFC 5480, PEM)
 depends_on:MBEDTLS_PEM_PARSE_C:MBEDTLS_ECP_C:MBEDTLS_ECP_DP_SECP192R1_ENABLED
 pk_parse_public_keyfile_ec:"data_files/ec_pub.pem":0
 
+Parse Public EC Key #2a (RFC 5480, PEM, secp192r1, compressed)
+depends_on:MBEDTLS_PEM_PARSE_C:MBEDTLS_ECP_C:MBEDTLS_ECP_DP_SECP192R1_ENABLED
+pk_parse_public_keyfile_ec:"data_files/ec_pub.comp.pem":0
+
 Parse Public EC Key #3 (RFC 5480, secp224r1)
 depends_on:MBEDTLS_PEM_PARSE_C:MBEDTLS_ECP_C:MBEDTLS_ECP_DP_SECP224R1_ENABLED
 pk_parse_public_keyfile_ec:"data_files/ec_224_pub.pem":0
 
+# Compressed points parsing does not support MBEDTLS_ECP_DP_SECP224R1 and
```

Comment:
Where is this limitation documented? I can't find anything about it in `ecp.h`.

Reply:
Good point again. I'm afraid right now it's only explicitly documented in [the ChangeLog entry](https://github.com/Mbed-TLS/mbedtls/pull/6282/files#diff-3b9d707a71bac75fb3ee1fb8f9bfce78c237752dee275586090126bee530b5e5R1) (and implicitly in the comments of `mbedtls_ecp_sw_derive_y` in `ecp.c`) which is not great.

I think we should fix that by adding something to the doxygen documentation of `mbedtls_ecp_point_read_binary()` (along the lines of the ChangeLog entry) and then ideally referencing it in its various callers. Wdyt? Where would you expect to find this documented?

Reply:
I would expect this to be documented for functions whose primary goal is to parse a point or an ECC public key: `mbedtls_ecp_point_read_binary`, `mbedtls_pk_parse_public_key`, and also in the documentation of `MBEDTLS_ECP_PF_COMPRESSED`. This can be by reference (e.g. “For compressed points, see `#MBEDTLS_ECP_PF_COMPRESSED` for limitations.”)